docs/Hooks.rst

```diff
@@ -132,6 +137,133 @@ In hooks, you have access to the full functionality provided by the EasyBuild fr
 so do ``import`` from ``easybuild.tools.*`` (or other ``easybuild.*`` namespaces) to leverage
 those functions.
 
+.. _hooks_parse_hook:
+
+Parse hook specifics
+++++++++++++++++++++
+
+``parse_hook`` is triggered right *after* reading the easyconfig file,
+before further parsing of some easyconfig parameters (like ``*dependencies``) into
+custom data structures is done.
+
+This is important since it allows to dynamically modify easyconfig files
+while they are still "raw", i.e., when the easyconfig parameter values are
+still basic Python data structures like lists, dictionaries, etc.
+that are easy to manipulate (see also :ref:`hooks_caveats_manipulating`).
+
+In ``parse_hook`` easyconfig parameters can be accessed and/or modified in a straightforward way,
+see :ref:`hooks_examples_openmpi_configopts`.
```

Comment:
This example is not the parse hook example, should point to the correct example instead.